### PR TITLE
Resolve JBUGs related to template string in testParser.bal

### PR DESCRIPTION
**Pull request by KavinduZoysa**
## Purpose
Related to #550

### CODE REVIEW

**Comment by KavinduZoysa** [comment]
Create new issue:
https://github.com/ballerina-platform/ballerina-lang/issues/33390

File: compiler/modules/front.syntax/tests/testParser.bal [hunk 1]
```diff
@@ -160,23 +160,22 @@ function invalidTokenSourceFragments() returns map<TokenizerTestCase>|error {
...
-         // JBUG #31444 #31713 #31717 can't write below literals in a simpler way
-        ["E", "\"\\" + "u{}\""],
+        ["OE", string`\`],
+        ["OE", string`"\n"`],
+        ["OE", string`"\r"`],
+        ["E", string`"\n"`],
+        ["E", string`"\r"`],
+        ["E", string`"\\`],
+        ["E", string`"\u{}"`],
+        // JBUG #33390
```
Comment (jclark) [inline compiler/modules/front.syntax/tests/testParser.bal]: Please say what the bug is

**Review by jclark — CHANGES_REQUESTED** [review]

File: compiler/modules/front.syntax/tests/testParser.bal (hunk 1 above)
Comment (KavinduZoysa) [reply compiler/modules/front.syntax/tests/testParser.bal]: Edited

**Review by jclark — APPROVED** [review]

File: compiler/modules/front.syntax/tests/testParser.bal [hunk 2]
```diff
@@ -160,23 +160,22 @@ function invalidTokenSourceFragments() returns map<TokenizerTestCase>|error {
...
-        ["E", "\"\n\""],
-        ["E", "\"\r\""],
-        ["E", "\"\\"],
-         // JBUG #31444 #31713 #31717 can't write below literals in a simpler way
-        ["E", "\"\\" + "u{}\""],
+        ["OE", string`\`],
+        ["OE", string`"\n"`],
+        ["OE", string`"\r"`],
+        ["E", string`"\n"`],
+        ["E", string`"\r"`],
```
Comment (manuranga) [inline compiler/modules/front.syntax/tests/testParser.bal:167]: This is meant to say strings with new lines are invalid, current translation is wrong.

**Pull request merged**